server/svix-server/src/core/security.rs

```diff
@@ -60,6 +61,9 @@ impl Permissions {
 pub struct CustomClaim {
     #[serde(rename = "org", default, skip_serializing_if = "Option::is_none")]
     pub organization: Option<String>,
+
+    #[serde(rename = "evt_ff", default, skip_serializing_if = "Option::is_none")]
```

Comment:
It's not just an event type feature flag, so `evt_ff` is very misleading.

I'd literally just call it: `feature_flags`. Yeah, it's maybe a bit longer (by a few chars), though it'll only be used by ones that actually have it, so it's a not-that-important optimization.

Reply:
Renamed to `feature_flags`